src/utils/withWidth.js

```diff
@@ -102,7 +107,13 @@ const withWidth = (options = {}) => Component => {
       }
 
       return (
-        <EventListener target="window" onResize={this.handleResize}>
+        <EventListener
+          target="window"
+          onResize={this.handleResize}
+          ref={node => {
+            this.mountNode = node;
```

Comment:
What do you mean?

Reply:
`this.mountNode` will be a reference to the `EventListener` instance. I am then doing `ownerDocument(this.mountNode)` which is wrong since `EventListener` will never have an `ownerDocument` property on it.